.storybook/webpack.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = ({ config, mode }) => {
+  config.module.rules.push({
+    test: /\.(ts|tsx)$/,
+    loader: require.resolve('babel-loader'),
+    options: {
+      presets: [['react-app', { flow: false, typescript: true }]],
+    },
+  });
+  config.resolve.extensions.push('.ts', '.tsx');
+  return config;
+};
```

Comment:
Custom `webpack` configuration for Storybook. This just runs `babel-loader` such that we strip off all the type information in our `.stories.tsx` files.